easybuild/easyconfigs/o/occt/occt-7.5.0p1-foss-2021a.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'occt'
+version = '7.5.0p1'
+
+homepage = 'https://www.opencascade.com/'
+description = """Open CASCADE Technology (OCCT) is an object-oriented C++
+class library designed for rapid production of sophisticated domain-specific
+CAD/CAM/CAE applications."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://files.salome-platform.org/Salome/other']
+sources = ['opencascade-%(version)s.tar.gz']
+checksums = ['c0bbe42f0c41b6dcd6a39a83961ca7e414eb2d5d919f4dcb930096f2a423a578']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('Mesa', '21.1.1'),
+    ('freetype', '2.10.4'),
+    ('Tcl', '8.6.11'),
+    ('Tk', '8.6.11'),
+    ('FreeImage', '3.18.0'),
+    ('tbb', '2020.3'),
+]
+
+separate_build_dir = True
+
+configopts = "-DUSE_FREEIMAGE=ON -D3RDPARTY_FREEIMAGE_DIR=$EBROOTFREEIMAGE "
+configopts += "-D3RDPARTY_TBB_DIR=$EBROOTTBB "
+configopts += "-D3RDPARTY_TCL_DIR=$EBROOTTCL "
+configopts += "-D3RDPARTY_TK_DIR=$EBROOTTK "
+configopts += "-D3RDPARTY_FREETYPE_DIR=$EBROOTFREETYPE "
+
+sanity_check_paths = {
+    'files': ['bin/DRAWEXE', 'bin/env.sh'],
+    'dirs': ['lib/cmake/opencascade'],
+}
+
```

I'm adding a simple sanity check command for `occt`, as we would like to have that for new contributions:
```suggestion

sanity_check_commands = ['DRAWEXE -h']

```